tests/integration_tests/datasets/api_tests.py

```diff
@@ -1711,12 +1712,15 @@ def test_bulk_delete_dataset_items(self):
         assert rv.status_code == 200
         expected_response = {"message": f"Deleted {len(datasets)} datasets"}
         assert data == expected_response
+        deleted_datasets = datasets
         datasets = (
             db.session.query(SqlaTable)
             .filter(SqlaTable.table_name.in_(self.fixture_tables_names))
             .all()
         )
         assert datasets == []
+        for dataset in deleted_datasets:
+            setattr(dataset, "_deleted", True)
```

Comment:
This test started failing since the cleanup was trying to delete already-deleted datasets.  This was fine for some reason before with the bulk-deletes.